pkg/ovs/daemonset.go

```diff
@@ -169,13 +145,14 @@ func DaemonSet(
 							TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
 						}, {
 							// ovn-controller container
+							// NOTE(slaweq): for some reason, when ovn-controller is started without
+							// bash shell, it fails with error "unrecognized option --pidfile"
```

Comment:
this is probably because you pass a single string that defines the whole CLI command; I think if you pass an array of strings, it may work. Worth a try.